hips/utils/tests/test_wcs.py

```diff
@@ -1,18 +1,35 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from numpy.testing import assert_allclose
-from ..testing import make_test_wcs_geometry
-
+from astropy.coordinates import SkyCoord
+from ..wcs import WCSGeometry
 
 class TestWCSGeometry:
     def setup(self):
-        self.wcs_geometry = make_test_wcs_geometry(case=0)
+        self.wcs_geometry = WCSGeometry.create_simple(
+            skydir=SkyCoord(0, 0, unit='deg', frame='galactic'),
+            height=1000, width=2000, fov="3 deg",
+            coordsys='galactic', projection='AIT'
+        )
 
-    def test_center(self):
+    def test_galactic_frame_center(self):
         c = self.wcs_geometry.center_skycoord
+
         assert c.frame.name == 'galactic'
-        assert_allclose(c.l.deg, 2, atol=1e-2)
-        assert_allclose(c.b.deg, 4.5, atol=1e-2)
+        assert_allclose(c.l.deg, 359.24, atol=1e-2)
+        assert_allclose(c.b.deg, -0.74, atol=1e-2)
+        assert_allclose(self.wcs_geometry.wcs.wcs.crpix, [500.,  1000.])
+        assert_allclose(self.wcs_geometry.wcs.wcs.cdelt, [-0.0015, 0.0015])
 
     def test_celestial_frame(self):
-        pass
-        # TODO (Adeel): add a test
+        wcs_geometry = WCSGeometry.create_simple(
+            skydir=SkyCoord(0, 0, unit='deg', frame='icrs'),
+            height=1000, width=2000, fov="3 deg",
+            coordsys='icrs', projection='AIT'
+        )
+        c = wcs_geometry.center_skycoord
+
+        assert c.frame.name == 'icrs'
+        assert_allclose(c.ra.deg, 359.24, atol=1e-2)
+        assert_allclose(c.dec.deg, -0.74, atol=1e-2)
+        assert_allclose(self.wcs_geometry.wcs.wcs.crpix, [500.,  1000.])
```

Comment:
I just tried reversing `crpix` values i.e.

geometry = WCSGeometry.create(skydir=SkyCoord(0, 0, unit='deg', frame='galactic'), width=2000, height=1000, coordsys='galactic', projection='AIT', cdelt=0.01, crpix=(1000, 500))

But I still get the same result as above.